wakuv2/nwaku.go

```
@@ -2642,6 +2660,37 @@ func (n *WakuNode) RelaySubscribe(pubsubTopic string) error {
 	return errors.New(errMsg)
 }
 
+func (n *WakuNode) RelayAddProtectedShard(clusterId uint16, shardId uint16, pubkey *ecdsa.PublicKey) error {
+	if pubkey == nil {
+		return nil // Nothing to do here
```

Isn't it better to return an error in this case?